blmath/optimization/objectives/normals.py

```diff
@@ -0,0 +1,242 @@
+# pylint: disable=arguments-differ, invalid-unary-operand-type
+import numpy as np
+import chumpy as ch
+import scipy.sparse as sp
+from blmath.numerics.matlab import row, col
+
+try:
+    from opendr.geometry import TriNormals, VertNormals, TriNormalsScaled, NormalizedNx3  # This lets us import bodylabs.geometry.normals.NormalizedNx3 etc. whether or not opendr is available. pylint: disable=unused-import
```

Comment:
How do you feel about this? It was originally done because we needed these things from opendr in some of the windows stuff for the army. I'd be inclined to leave these here and import them from here in opendr...